web/src/views/DandisetLandingView/ListingComponent.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div>
+    <template v-if="schema.type === 'array'">
+      <template v-if="schema.items.type === 'object'">
+        <v-expansion-panels>
+          <v-expansion-panel
+            v-for="item in data"
+            :key="item[schema.items.listingKey]"
+          >
+            <!-- item is an object -->
+            <v-expansion-panel-header>{{ item[schema.items.listingKey] }}</v-expansion-panel-header>
+            <v-expansion-panel-content>
+              <v-list>
+                <v-list-item
+                  v-for="(value, key) in item"
+                  :key="key"
+                >
+                  <!-- value's type matches that specified at schema.items.properties[key].type -->
+                  <ListingComponent
+                    :schema="schema.items.properties[key]"
+                    :data="value"
+                  />
+                </v-list-item>
+              </v-list>
+            </v-expansion-panel-content>
+          </v-expansion-panel>
+        </v-expansion-panels>
+      </template>
+      <template v-else>
+        <b>{{ schema.title }}</b>: {{ data.join(', ') }}
```

Comment:
I assume an array of arrays doesn't exist in the schema?

Reply:
It seems like `ethicsApprovals` might, so I'll add the case. It shouldn't be too hard.

Reply:
I couldn't find any examples of an array within an array, so I just added a placeholder.